pkg/instrumentation/auto/restart.go

```diff
@@ -0,0 +1,44 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package auto
+
+import (
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/aws/amazon-cloudwatch-agent-operator/pkg/instrumentation"
+)
+
+const (
+	restartedAtAnnotation = "kubectl.kubernetes.io/restartedAt"
```

Comment:
Do we want to re-use the kubectl annotation or use a new one that conveys our operator restarted it?
Some customers that never use kubectl on say prod clusters may find it odd to see this pop up.

Reply:
Yeah makes sense. I'll change it.